src/main/kotlin/no/nb/bikube/core/controller/CoreController.kt

```diff
@@ -59,25 +58,104 @@ class CoreController (
+                    else -> throw NotSupportedException("Material type $materialType is not supported.")
+                }
+            }
+            SearchType.LOCATION -> {
+                when(materialTypeToCatalogueName(materialType)) {
+                    CatalogueName.COLLECTIONS -> ResponseEntity.ok(axiellService.searchPublisherPlaceByName(searchTerm))
+                    else -> throw NotSupportedException("Material type $materialType is not supported.")
+                }
+            }
+            else -> throw NotSupportedException("Search type $searchType is not supported.")
+        }
+    }
+
+    @PostMapping("/publisher", produces = [MediaType.APPLICATION_JSON_VALUE])
+    @Operation(summary = "Create new publisher")
+    @ApiResponses(value = [
+        ApiResponse(responseCode = "200", description = "OK"),
+        ApiResponse(responseCode = "400", description = "Bad request"),
+        ApiResponse(responseCode = "500", description = "Server error")
+    ])
+    fun createPublisher(
+        @RequestBody publisher: String
+    ): Mono<ResponseEntity<Publisher>> {
+        return if (publisher.isEmpty()) {
+            throw BadRequestBodyException("Publisher cannot be empty.")
+        } else axiellService.createPublisher(publisher).map { created ->
+            ResponseEntity.ok(created)
+        }
+    }
+
+    @PostMapping("/publisher-place", produces = [MediaType.APPLICATION_JSON_VALUE])
+    @Operation(summary = "Create new publisher place (location)")
+    @ApiResponses(value = [
+        ApiResponse(responseCode = "200", description = "OK"),
+        ApiResponse(responseCode = "400", description = "Bad request"),
+        ApiResponse(responseCode = "500", description = "Server error")
+    ])
+    fun createPublisherPlace(
+        @RequestBody location: String
+    ): Mono<ResponseEntity<PublisherPlace>> {
+        return if (location.isEmpty()) {
+            throw BadRequestBodyException("Publisher place (location) cannot be empty.")
+        } else axiellService.createPublisherPlace(location).map { created ->
+            ResponseEntity.ok(created)
+        }
+    }
+
+    @PostMapping("/language", produces = [MediaType.APPLICATION_JSON_VALUE])
+    @Operation(summary = "Create new ISO-639-2 language code")
+    @ApiResponses(value = [
+        ApiResponse(responseCode = "200", description = "OK"),
+        ApiResponse(responseCode = "400", description = "Bad request"),
+        ApiResponse(responseCode = "500", description = "Server error")
+    ])
+    fun createLanguage(
+        @RequestBody language: String
+    ): Mono<ResponseEntity<Language>> {
+        if (!Regex("^[a-z]{3}$").matches(language)) {
+            throw BadRequestBodyException("Language code must be a valid ISO-639-2 language code.")
+        }
+        return axiellService.createLanguage(language).map { created ->
+            ResponseEntity.ok(created)
         }
     }
```

Comment:
Do we really need these as endpoints, or is it enough with service methods when needed? I see no reason a user/application should create these without registering a title/item.